packages/utilities/tsconfig.json

```diff
@@ -7,11 +7,11 @@
     "jsx": "react",
     "declaration": true,
     "sourceMap": true,
+    "strict": true,
```

Comment:
Since `@fluentui/utilities` no longer needs to be included in converged compilation, I'd prefer to revert these settings and the related changes.

Reply:
I don't think that we should throw out any work, but at the same it's not anymore required for this PR 🤔 

I propose to apply this changes in a separate PR (really want to have more "strict" packages ❤️ ). @ecraig12345 @Hotell what do you think?

Reply:
The main reason I was proposing reverting for now was in favor of a more detailed pass through to enable the option later (if we decide to invest in that). This would involve looking through the files and where possible, trying to understand the correct typing based on usage (and fix any missing null checks etc), rather than making the quickest change to silence the error.

I also very very strongly prefer to avoid `@ts-ignore` whenever possible since it's a "nuclear option": it turns off *all type checking* for the next line, rather than just the part where the error is coming from, opening up to more errors in the future.

Reply:
you already spent time reviewing those changes, there are only 2 ts-ignores documented ( Im gonna create issue and ref it there as well) which don't affect any public facing API. 

I explicitly checked those `!` non constructor assignment in affected files, to verify if that's good enough solution for now.

To summarise, while I can create another PR , I don't think it's necessary (reasoning provided). As a side note/thought, this PR is open for almost 2 weeks which is blocking the whole improved DX initiative for convergence ✌️